app/src/main/java/net/opendasharchive/openarchive/services/dropbox/DropboxLoginActivity.kt

```diff
@@ -118,11 +118,11 @@ class DropboxLoginActivity : AppCompatActivity() {
             // There was an error; don't attempt login and focus the first
             // form field with an error.
             focusView?.requestFocus()
-        } else {
-            // Show a progress spinner, and kick off a background task to
-            // perform the user login attempt.
-            Auth.startOAuth2Authentication(this@DropboxLoginActivity, getString(R.string.dropbox_key))
-        }
+            } else {
```

Comment:
is this a whitespace convention I'm too python-ey to understand?

Reply:
Ahh my studio is configured for Flutter. I just did a format on this file as per the recommended Android standards and pushed out the change. Should look better now.